requirements.txt

```diff
@@ -1,31 +1,34 @@
-basicsr
-diffusers
+basicsr==1.4.2

```

Comment:
We should be using exact versions for these libs so that we limit any future breaking changes/bugs.